### ISSUE TITLE
Switch off html5 validation

### ISSUE THREAD

**Issue by markconroy** [localgovdrupal/localgov_core#261]
HTML 5 default browser validation is not up to the same standard that Drupal's validation has. 

Drupal accessibility maintainers suggest it should be turned off and that we should use Drupal's form validation instead.

This would mean instead of a very simple form validation with error reports like this:

![Screenshot 2024-12-02 at 14 49 46](https://github.com/user-attachments/assets/bcd6e1fa-d0d2-4a30-8634-f0a7f17a8f6c)

We would have more comprehensive form validation with error reports like this:

![Screenshot 2024-12-02 at 14 49 54](https://github.com/user-attachments/assets/0741c282-0d81-4522-ad90-012b7c8e3f0c)
